deny.toml

```diff
@@ -69,7 +69,7 @@ ignore = [
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
 # The lint level for crates which do not have a detectable license
-unlicensed = "deny"
+unlicensed = "allow"
```

Comment:
I'd rather keep the lint and whitelist a package if it's really needed, is that possible?